### DISCUSSION TITLE
Reload all Tabs in a Group

### DISCUSSION THREAD

**Discussion by thegrobi** [mbnuqw/sidebery#1528] [Q&A]
I've been testing Sidebery for a day.
I have created about 10 groups: Sport, Buy, Kodi etc.
There are 3-15 tabs in each group.

When I click on a group name, I want to update all the tabs in it with a "reload all tabs" command.
How can I do this without having to select all tabs first?

**Comment by mbnuqw** [comment]
Related feature request: #460
Workaround: You can right-click on group icon - this will select the whole branch and open context menu where you can click on the "reload option". Although, this will reload the group (parent) page.

**Reply by thegrobi** [reply]
Thank you. That is great.
But it is not self-explanatory.

**Comment by thegrobi** [comment]
I just found another method:
I have created a command in the mouse settings:
"Long left mouse click reloads tab".

When the group is closed, I can reload all the tabs placed in it with a long mouse click.
But not when the group is closed?!
